packages/webview_flutter/webview_flutter_android/lib/src/legacy/webview_android_widget.dart

```diff
@@ -9,24 +9,25 @@ import 'package:flutter/widgets.dart';
 // ignore: implementation_imports
 import 'package:webview_flutter_platform_interface/src/webview_flutter_platform_interface_legacy.dart';
 
-import '../android_webview.dart' as android_webview;
+import '../android_webkit.g.dart' as android_webview;
+import '../android_webkit_constants.dart';
 import '../weak_reference_utils.dart';
 import 'webview_android_cookie_manager.dart';
 
 /// Creates a [Widget] with a [android_webview.WebView].
 class WebViewAndroidWidget extends StatefulWidget {
   /// Constructs a [WebViewAndroidWidget].
-  const WebViewAndroidWidget({
+  WebViewAndroidWidget({
```

Comment:
This file isn't exported by `lib/src/webview_flutter_android_legacy` so this shouldn't be a breaking change.